api/admin/oauth.py

```diff
@@ -47,7 +47,15 @@ def callback(self, request={}):
             ), redirect_url
 
     def google_error_problem_detail(self, error):
-        error_detail = GOOGLE_OAUTH_FAILURE.detail + _("Error: ") + error
+        error_detail = _("Error: %(error)s", error=error)
+
+        # ProblemDetail.detailed requires the detail to be an internationalized
+        # string, so pass the combined string through _ as well even though the
+        # components were translated already. Space is a variable so it doesn't
+        # end up in the translation template.
+        space = " "
```

Comment:
Will this cause an error when the already-localized string can't be further localized?